src/attributes.jl

```diff
@@ -1138,6 +1138,18 @@ indices of type `F`-in-`S` in the model (i.e., of length equal to the value of
 """
 struct ListOfConstraintIndices{F,S} <: AbstractModelAttribute end
 
+function get_fallback(
+    model::ModelLike,
+    ::ListOfConstraintIndices{F,S},
+) where {F,S}
+    if supports_constraint(model, F, S) ||
```

Comment:
```suggestion
    # Throw error here so that we don't return an incorrect value if solvers
    # don't implement this method for some constraints they support.
    if supports_constraint(model, F, S) ||
```